tests/check_new_syntax.py

```diff
@@ -1,12 +1,19 @@
 #!/usr/bin/env python3
 
 import ast
+import re
 import sys
 from itertools import chain
 from pathlib import Path
 
+SINGLE_LINE_COLLECTIONS_SET_IMPORT_REGEX = re.compile(r"from\s+collections\.abc\s+import\s+(\w+\s*,\s*)*Set")
+# This one won't match unless the code's been formatted with black.
+MULTI_LINE_COLLECTIONS_SET_IMPORT_REGEX = re.compile(
+    r"from\s+collections\.abc\s+import\s*\((\n    \w+,\s*)*\n    Set(\s*,\s*\n    \w+)*\s*,?\s*\)"
+)
```

Comment:
It would be better to add a `visit_Import()` method to `OldSyntaxFinder`.

Reply:
Fair, I'm more confident using regular expressions which is why I went that route, but I'll give it a go.

Reply:
Actually, I think the correct name is `visit_ImportFrom`.

```
>>> import ast
>>> ast.parse('from collections.abc import Set')
<_ast.Module object at 0x7f8cae6ae630>
>>> print(ast.dump(_))
Module(body=[ImportFrom(module='collections.abc', names=[alias(name='Set', asname=None)], level=0)])
>>> 
```

Reply:
How's the new version using AST? (You're... right, using AST is much easier!)